README.md

```diff
@@ -156,6 +156,31 @@ and add this config to your `package.json`:
 
 See https://github.com/okonet/lint-staged#configuration for more details about how you can configure 🚫💩 lint-staged.
 
+Alternately you can just save this script as `pre-commit` in your `.git/hooks` directory and give it execute permissions:
+
+```bash
+#!/bin/sh
+# don't allow the commit if it contains any files that are not correctly
```

Comment:
Can you remove all those comments. This is not very helpful for people to copy and paste that. Also, for the inspiration, I think you should put it in the commit message and pull request but doesn't need to be on the README.